rskj-core/src/main/java/co/rsk/mine/MinerUtils.java

```diff
@@ -170,24 +170,24 @@
                 }
 
                 if (!(tx instanceof RemascTransaction) && tx.getGasPriceAsInteger().compareTo(minGasPrice) < 0) {
-                    logger.warn("Rejected transaction {} because of low gas account {}, removing tx from pending state.", Hex.toHexString(tx.getHash()), txSender);
+                    logger.warn("Rejected tx={} because of low gas account {}, removing tx from pending state.", hexHash, txSender);
 
                     txsToRemove.add(tx);
                     continue;
                 }
 
                 if (!expectedNonce.equals(txNonce)) {
-                    logger.warn("Invalid nonce, expected {}, found {}, tx {}", expectedNonce.toString(), txNonce.toString(), Hex.toHexString(tx.getHash()));
+                    logger.warn("Invalid nonce, expected {}, found {}, tx={}", expectedNonce, txNonce, hexHash);
                     continue;
                 }
 
                 accountNonces.put(txSender, txNonce);
 
-                logger.info("Accepted transaction {} sender: {} value: {} nonce: {}", Hex.toHexString(tx.getHash()), txSender, Hex.toHexString(tx.getValue()), Hex.toHexString(tx.getNonce()));
+                logger.debug("Accepted tx={} sender: {} value: {} nonce: {}", hexHash, txSender, hexValue, hexNonce);
             } catch (Exception e) {
                 // Txs that can't be selected by any reason should be removed from pending state
-                String hash = null == tx.getHash() ? "" : Hex.toHexString(tx.getHash());
-                logger.warn("Error when processing transaction: " + hash, e);
+                String hash = null == tx.getHash() ? "" : Hex.toHexString(tx.getHash());;
+                logger.warn(String.format("Error when processing tx=%s" + hash), e);
```

Comment:
there's a bug here. you're concatenating instead of formatting.

Reply:
:+1: